src/UnderlineNav2/UnderlineNav.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import {EyeIcon} from '@primer/octicons-react'
+import {Meta} from '@storybook/react'
+import UnderlineNav, {UnderlineNavProps} from './index'
+import {BaseStyles, ThemeProvider} from '..'
+import {within, userEvent} from '@storybook/testing-library'
+import {expect} from '@storybook/jest'
+
+export default {
+  title: 'Layout/UnderlineNav',
+  argTypes: {
+    align: {
+      defaultValue: 'left',
+      control: {
+        type: 'radio',
+        options: ['left', 'right']
+      }
+    }
+  },
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ]
+} as Meta
+
+export const DefaultNav = (args: UnderlineNavProps) => {
+  return (
+    <UnderlineNav {...args}>
+      <UnderlineNav.Link selected>Item 1</UnderlineNav.Link>
+      <UnderlineNav.Link>Item 2</UnderlineNav.Link>
+      <UnderlineNav.Link>Item 3</UnderlineNav.Link>
+    </UnderlineNav>
+  )
+}
+
+// Testing selection
+DefaultNav.play = async ({canvasElement}: {canvasElement: HTMLElement}) => {
+  const canvas = within(canvasElement)
+  const item = await canvas.getByText('Item 1')
+  const aElem = await item.closest('a')
+  const item2 = await canvas.getByText('Item 2')
+  const aElem2 = await item2.closest('a')
+  userEvent.click(aElem2 as Element)
+  expect(aElem2).toHaveAttribute('aria-current', 'page')
+  userEvent.click(aElem as Element)
+  expect(aElem).toHaveAttribute('aria-current', 'page')
+}
+
+export const withIcons = (args: UnderlineNavProps) => {
+  return (
+    <UnderlineNav {...args}>
+      <UnderlineNav.Link selected leadingIcon={EyeIcon}>
+        Item 1
+      </UnderlineNav.Link>
+      <UnderlineNav.Link>Item 2</UnderlineNav.Link>
+    </UnderlineNav>
+  )
+}
+
+export const rightAlign = (args: UnderlineNavProps) => {
```

Comment:
I don't think we need this story since we can change the alignment in the controls.

Reply:
Well.. I'd put it in docs. so I thought I'd have a story for it as well.